distribution/lib/Standard/Base/0.0.0-dev/src/Data/Set.enso

```diff
@@ -47,12 +47,17 @@ type Set
     to_vector : Vector
     to_vector self = self.underlying_map.keys
 
-    ## GROUP Metadata
-       ICON metadata
+    ## ICON metadata
        Returns the number of elements in this set.
     size : Integer
     size self = self.underlying_map.size
 
+    ## GROUP Metadata
+       ICON metadata
+       Returns the number of entries in this set.
```

Comment:
Can we make the descriptions the same? I had to think if there was a difference between elements and entries

Reply:
Done.